pkg/storkctl/migrationschedule.go

```diff
@@ -578,13 +578,17 @@ func getMigrationNamespaces(namespaceList []string, namespaceSelectors map[strin
 	for _, ns := range namespaceList {
 		uniqueNamespaces[ns] = true
 	}
-	labelSelectorNamespaces, err := core.Instance().ListNamespaces(namespaceSelectors)
-	if err != nil {
-		return nil, err
-	}
-	for _, ns := range labelSelectorNamespaces.Items {
-		uniqueNamespaces[ns.GetName()] = true
+
+	for key, val := range namespaceSelectors {
+		namespaces, err := core.Instance().ListNamespaces(map[string]string{key: val})
+		if err != nil {
+			return nil, err
```

Comment:
To enter this branch I will have to mock the ListNamespaces method to return an error. So habe not taken this up for now.

Rest of the codecov warnings are not related to changes made in this PR and have already justified why they aren't covered by UTs in their respective PRs.